src/test/resources/test-data/factors-response.json

```diff
@@ -120,6 +120,7 @@
     "reason": "custom_rule"
```

Comment:
Could you also update `full-request.json` with the `session_age` and `session_id`?